include/radix/data/model/MeshLoader.hpp

```diff
@@ -1,8 +1,8 @@
 #ifndef RADIX_MESHLOADER_HPP
 #define RADIX_MESHLOADER_HPP
 
-#include <string>
 #include <map>
+#include <string>
 
 #include <radix/data/model/Mesh.hpp>
```

Comment:
If you look at other source files, we like to include from radix first, and then external. I know this isn't documented anywhere, it probably should be, but just so you know ^^